telegram-service/src/app.update.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Command,
+  Ctx,
+  Hears,
+  Start,
+  Update,
+  Sender,
+  Action,
+} from 'nestjs-telegraf';
+import { Context } from './interfaces/context.interface';
+import { SceneNames, Actions, Commands } from './app.constants';
+
+@Update()
+export class AppUpdate {
+  supportedCommands = [
+    { command: '/help', description: 'Help' },
+    { command: '/start', description: 'Start' },
+  ];
+
+  @Start()
+  async onStart(ctx: Context): Promise<void> {
+    await ctx.reply(
+      "Hello, I'm the bid bot! Here is a list of commands that I can help you with",
+      {
+        reply_markup: {
+          inline_keyboard: [
+            [
+              { text: '🤑 Make a bid', callback_data: Actions.BID },
+              { text: '⏱️ History', callback_data: 'history' },
+            ],
+            [{ text: '📢 Help', callback_data: Commands.HELP }],
+            [{ text: '🥇 Rate', callback_data: 'rate' }],
+            [
+              {
+                text: '🤓 Open GitHub',
+                url: 'https://github.com/Gabriellji/bit_bot',
```

Comment:
move url to some const, you might want to change name of your repo after some time

Reply:
+

Reply:
> move url to some const, you might want to change name of your repo after some time

++